docs/source/quickstart.rst

```diff
@@ -91,187 +113,206 @@ as follows (this example is also included in ``quickstart/mlflow_tracking.py``):
+.. code-section::
 
-which returns::
+    .. code-block:: shell
 
-    [1, 0]
+        databricks configure
 
-For more information, see :doc:`models`.
+Set the ``Databricks Host`` to the URL of your Databricks workspace, and set the ``Username`` and ``Password`` to the credentials you use to access the workspace. If you've created an authentication token for your Databricks workspace (``databricks tokens create``), you can use it instead of your password. Call ``databricks configure`` with the ``-t, \--token`` option.
 
+..
+    Consider adding a link to the Databricks CLI docs for tokens create
 
-.. _quickstart_logging_to_remote_server:
+In your training code, modify the call to ``mlflow.set_tracking_uri`` to use Databricks and set the experiment to the path of your experiment in Databricks, replacing ``user_name`` and ``experiment_name`` with the appropriate values:
 
-Logging to a Remote Tracking Server
------------------------------------
-In the examples above, MLflow logs data to the local filesystem of the machine it's running on.
-To manage results centrally or share them across a team, you can configure MLflow to log to a remote
-tracking server. To get access to a remote tracking server:
+.. code-section::
+
+    .. code-block:: python
+
+        mlflow.set_tracking_uri("databricks")
+        mlflow.set_experiment(f"/Users/{user_name}/{experiment_name}")
+
+If the specified experiment does not exist, it will be created.
+
+Store a model in MLflow
+-------------------------
 
-Launch a Tracking Server on a Remote Machine
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-:ref:`Launch a tracking server <tracking_server>` on a remote machine.
+An MLflow Model is a directory that packages machine learning models and support files in a standard format. The directory contains:
 
-You can then :ref:`log to the remote tracking server <logging_to_a_tracking_server>` by
-setting the ``MLFLOW_TRACKING_URI`` environment variable to your server's URI, or
-by adding the following to the start of your program:
+- An **MLModel** file in YAML format specifying the model's **flavor** (or **flavors**), dependencies, signature (if supplied), and important metadata;
+- The various files required by the model's flavor(s) to instantiate the model. This will often be a serialized Python object;
+- Files necessary for recreating the model's runtime environment (for instance, a **conda.yaml** file); and
+- Optionally, an input sample
 
-  .. code-section::
+When using autologging, MLflow will automatically log the run's model. You can also log a model manually by calling ``mlflow.{library_module_name}.log_model``. In addition, if you wish to load the model soon, it may be convenient to output the run's ID directly to the console. For that, you'll need the object of type ``mlflow.ActiveRun`` for the current run. You get that object by wrapping all of your logging code in a ``with mlflow.start_run() as run:`` block. For example:
```

Comment:
Nit: I think it's a bit unclear what "run's model" means; a run could have multiple models or no models at all

```suggestion
When using autologging, MLflow will automatically log models to the runs that it creates. You can also log a model manually by calling ``mlflow.{library_module_name}.log_model``. In addition, if you wish to load the model soon, it may be convenient to output the run's ID directly to the console. For that, you'll need the object of type ``mlflow.ActiveRun`` for the current run. You get that object by wrapping all of your logging code in a ``with mlflow.start_run() as run:`` block. For example:
```

Reply:
Nit: Can we link to the Python API docs for `start_run()` and `ActiveRun`?

Reply:
DONE